File: tests/nodejs/cases/case_085_route_ticket.js
/**
 * Task case 085: route_ticket
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case085RouteTicket(input) {
  // Task: route ticket records and prepare transport-ready payload.
  const taskName = 'route_ticket';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'ticket',
    stage: 'route',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  // SAFE: payload remains local; only primitive checksum string is returned.
  return payload.checksum;
}

module.exports = {
  case085RouteTicket,
};

File: tests/python/cases/case_085_route_ticket.py
"""
Task case 085: route ticket payload for downstream business processing.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_085_route_ticket(input_data):
    # Task: route ticket records and prepare transport-ready payload.
    task_name = 'route_ticket'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'ticket',
        'stage': 'route',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    # SAFE: payload remains local; only primitive checksum string is returned.
    return payload['checksum']
